### PR TITLE
feat(server): Add reply count & latency metrics

### PR DESCRIPTION
**Pull request by chakaz**

### CODE REVIEW

File: src/facade/reply_builder.h [hunk 1]
```diff
@@ -155,6 +155,27 @@ class SinkReplyBuilder {
 
   virtual size_t UsedMemory() const;
 
+  enum SendStatsType {
+    kRegular,
+    kBatch,
+    kCount,
```
Comment (romange) [inline src/facade/reply_builder.h:161]: can you add a comment about each of them? 
I first thought `kCount` is another type, maybe `kNumTypes` ?

File: src/server/server_family.cc [hunk 2]
```diff
@@ -825,6 +826,40 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
...
   AppendMetricWithoutLabels("net_output_bytes_total", "", conn_stats.io_write_bytes,
                             MetricType::COUNTER, &resp->body());
+  {
+    string send_latency_metrics;
+    constexpr string_view kReplyLatency = "reply_latency_usec_total";
+    AppendMetricHeader(kReplyLatency, "Reply latency per mode", MetricType::COUNTER,
+                       &send_latency_metrics);
+
+    string send_count_metrics;
+    constexpr string_view kReplyCount = "reply_count";
```
Comment (romange) [inline src/server/server_family.cc:836]: I think the naming convention of both metrics not according to the prometheus standard. See https://prometheus.io/docs/practices/naming/

specifically, latency must be seconds and must be end with "duration_seconds" imho. You can also see how prometheus itself names its metrics by starting our monitoring stack and checking out `:9090/metrics`

Comment (chakaz) [reply src/server/server_family.cc:836]: That's a good resource, thanks for sharing!
(done)

File: src/facade/reply_builder.h (hunk 1 above)
Comment (chakaz) [reply src/facade/reply_builder.h:161]: done

**Review by romange — APPROVED** [review]

**Pull request merged**